larpandora/LArPandoraInterface/CMakeLists.txt

```diff
@@ -1,42 +1,50 @@
 include_directories( $ENV{PANDORA_INC} )
 include_directories( $ENV{LARPANDORACONTENT_INC} )
 
-art_make(
-          LIB_LIBRARIES larcorealg_Geometry
-                        larcore_Geometry_Geometry_service
-                        larsim_Simulation lardataobj_Simulation
-                        larsim_MCCheater_ParticleInventoryService_service
-                        lardataalg_DetectorInfo
-                        lardataobj_RawData
-                        lardataobj_RecoBase
-                        lardataobj_AnalysisBase
-                        lardata_ArtDataHelper
-                        lardata_Utilities
-                        larreco_RecoAlg_ClusterRecoUtil
-                        larreco_RecoAlg
-                        larreco_ClusterFinder
-                        larreco_Calorimetry
-                        ${PANDORASDK}
-                        ${PANDORAMONITORING}
-                        LArPandoraContent
-                        nusimdata_SimulationBase
-                        ${ART_FRAMEWORK_CORE}
-                        ${ART_FRAMEWORK_PRINCIPAL}
-                        ${ART_FRAMEWORK_SERVICES_REGISTRY}
-                        ${ART_FRAMEWORK_SERVICES_BASIC}
-                        ${ART_ROOT_IO_TFILE_SUPPORT} ROOT::Core
-                        ${ART_ROOT_IO_TFILESERVICE_SERVICE}
-                        art_Persistency_Common
-                        art_Persistency_Provenance
-                        art_Utilities
-                        canvas
-                        ${MF_MESSAGELOGGER}
-                        ${FHICLCPP}
-                        cetlib cetlib_except
-                        ROOT::Geom
-                        ${ROOT_BASIC_LIB_LIST}
-                        MODULE_LIBRARIES larpandora_LArPandoraInterface
-          )
+set(CORE_LIB_LIST
```

Comment:
I'm not an expert in the larsoft cmake setup, but this change from `art_make` to `set` is, I think, the recommended approach. Can check comments on recent larpandora/PandoraModularShower reco PR.

Reply:
This is just an intermediate step. Given that the build of the DL library is optional I opted to define a variable containing all of the standard libraries so that I could just reference this in the two versions of the art_make call below.